src/cppcache/src/TcrMessage.hpp

```diff
@@ -1170,7 +1170,7 @@ class TcrMessageHelper {
     }
     if (compId != expectedPartType) {
       char exMsg[256];
-      ACE_OS::snprintf(exMsg, 255,
+      std::snprintf(exMsg, 255,
```

Comment:
Is there a command line tool (e.g. lint) that I can run to ensure changes adhere to the style guide?  Also to be clear, it's the fact that the following lines are indented more than 2 spaces (than the line above) that is in violation?